SpatialGDK/Documentation/known-issues.md

```diff
@@ -1,28 +1,6 @@
 <%(TOC)%>
-# GDK for Unreal known issues
+# Known issues
```

Comment:
As we are changing the contents of this page, we need to add the update info at the bottom.
```
<br/>

------------
2019-04-25 Page updated with full editorial review </br>
2019-04-25 Removed out-dated known issues from this page and added links to current known issues.
<br/>

```